sync/class.jetpack-sync-module-updates.php

```diff
@@ -195,18 +199,43 @@ public function validate_update_change( $value, $expiration, $transient ) {
 		$checksums[ $transient ] = $new_checksum;
 
 		update_option( self::UPDATES_CHECKSUM_OPTION_NAME, $checksums );
-		/**
-		 * jetpack_{$transient}_change
-		 * jetpack_update_plugins_change
-		 * jetpack_update_themes_change
-		 * jetpack_update_core_change
-		 *
-		 * @since 5.1.0
-		 *
-		 * @param array containing info that tells us what needs updating
-		 *
-		 */
-		do_action( "jetpack_{$transient}_change", $value );
+		if ( 'update_core' === $transient ) {
+			/**
+			 * jetpack_update_core_change
+			 *
+			 * @since 5.1.0
+			 *
+			 * @param array containing info that tells us what needs updating
+			 *
+			 */
+			do_action( "jetpack_update_core_change", $value );
+			return;
+		}
+		if ( empty( $this->updates ) ) {
```

Comment:
```suggestion
		if ( ! empty( $this->updates ) ) {
```

Reply:
No we want to only add the 
`add_action( 'shutdown', array( $this, 'sync_last_event' ), 9 );

updates once.